pyriemann/transfer/methods.py

```diff
@@ -0,0 +1,127 @@
+import numpy as np
+from sklearn.model_selection import StratifiedShuffleSplit
+
+
+def encode_domains(X, y, domain):
+    r"""Encode the domains of the data points in the labels
+
+    We handle the possibility of having different domains for the datasets by
+    extending the labels of the data points and including this information to
+    them. For instance, if we have a datapoint x with class `left_hand` on the
+    `domain_01` then its extended label will be `left_hand/domain_01`. Note
+    that if the classes were integers at first, they will be converted to
+    strings.
+
+    Parameters
+    ----------
+    X : ndarray, shape (n_matrices, n_channels, n_channels)
+        Set of SPD matrices.
+    y : ndarray, shape (n_matrices,)
+        Labels for each matrix.
+    domain : ndarray, shape (n_matrices,)
+        Which domain each data point belongs to.
+
+    Returns
+    -------
+    X_enc : ndarray, shape (n_matrices, n_channels, n_channels)
+        The same set of SPD matrices given as input.
+    y_enc : ndarray, shape (n_matrices,)
+        Extended labels for each data point.
+    """
```

Comment:
Should be added in all functions/classes added in API.
```suggestion
    Notes
    -----
    .. versionadded:: 0.3.1
    """
```